### PR TITLE
✏️ Edits 'dumb' to add related words, more clarity

### LINKED ISSUE #365: Define "dumbfounded" & "dumbstruck"
Both of these build on the ableism inherent in "[dumb](https://www.selfdefined.app/definitions/dumb/)"
We could make these definitions children of that word.

Some alternatives:
astounded
awestruck
flabbergasted
nonplussed

Some quick research:
https://cishits.tumblr.com/post/79281810833/two-things-1-i-feel-like-ableist-language-is
https://www.quora.com/Do-you-know-when-you-are-using-derogatory-ableist-language/answer/Jayden-Brunner

### PR DESCRIPTION
**Pull request by tatianamac**
#365

## Overview
- Adds synonyms per #365 with additional resources
- Cleans up some of the language
- Adds additional suggested words

### CODE REVIEW

File: 11ty/definitions/dumb.md [hunk 1]
```diff
@@ -8,18 +8,30 @@ defined: true
...
+    - amazed
+    - awed
+    - floored
+    - nonplussed
+  dumbstruck (noun):
+    - astonished
+    - perplexed
+    - shocked
+    - stunned
+    - stupified
```
Comment (Primigenus) [inline 11ty/definitions/dumb.md]: I think stupefied carries the same ableist loaded meaning as dumbfounded, right?

File: 11ty/definitions/dumb.md [hunk 2]
```diff
@@ -8,18 +8,30 @@ defined: true
...
-  - senseless
-  - strange
-  - weird
-  - unfamiliar
-  - unfathomable
-  - unnecessary
+  dumb (noun):
+    - basic
+    - incomprehensible
+    - nonsensical
```
Comment (Primigenus) [inline 11ty/definitions/dumb.md]: Should we avoid recommending any language connecting senses or intelligence to lack of knowledge?

File: 11ty/definitions/dumb.md [hunk 3]
```diff
@@ -8,18 +8,30 @@ defined: true
...
+    - redundant
+    - shallow
+    - silly
+    - senseless
```
Comment (Primigenus) [inline 11ty/definitions/dumb.md]: Same as above comment

**Review by Primigenus — COMMENTED** [review]
Love this, thank you, just some minor questions around some alt suggestions. Apologies for not catching your request back in November!

File: 11ty/definitions/dumb.md (hunk 1 above)
Comment (tatianamac) [reply 11ty/definitions/dumb.md]: Good catch!

File: 11ty/definitions/dumb.md (hunk 2 above)
Comment (tatianamac) [reply 11ty/definitions/dumb.md]: I get what you're saying—my understanding is nonsense is connotes sense in the meaning of reason. From what I've read particularly from Deaf folks, it seems like there's a desire to not overcorrect (i.e., avoid words like "I see what you mean". With that said, I'd rather err on the side of caution so I can remove it, but wanted to provide the context!

**Comment by tatianamac** [comment]
@Primigenus Thank you for the thoughtful notes, Rahul! I pushed up some changes; let me know what you think—if good, will you approve? Thank you!

(Also no sorries!)

File: 11ty/definitions/dumb.md (hunk 2 above)
Comment (Primigenus) [reply 11ty/definitions/dumb.md]: Yeah, I've heard similar things from Deaf folks. It feels like maybe there's an heuristic here where we avoid using it as an adjective but are OK with it as a noun? Eg. "that's nonsense" seems alright to me, but "you're being nonsensical" implies that potentially ableist comparison. What do you think?

**Review by Primigenus — APPROVED** [review]
LGTM, thank you Tatiana!

File: 11ty/definitions/dumb.md (hunk 2 above)
Comment (tatianamac) [reply 11ty/definitions/dumb.md]: I can see that! Either way I agree we should remove!

**Review by Primigenus — APPROVED** [review]

**Pull request merged**